server/tests/test_auth.py

```diff
@@ -305,6 +308,25 @@ def test_auth_me_not_logged_in(client: FlaskClient):
     assert json.loads(rv.data) is None
 
 
+def test_session_expiration(client: FlaskClient, aa_email: str):
+    original_session_lifetime = app.permanent_session_lifetime
+    assert original_session_lifetime > timedelta(minutes=1)
+    assert app.config["SESSION_REFRESH_EACH_REQUEST"] is True
```

Comment:
where is this set? Does this mean tests fail if you don't have the right env var?

Reply:
this is a default flask config variable. i just added it here to make sure it didn't get changed, since we rely on its behavior. i can add an explanatory comment

Reply:
oh got it. Yeah a comment would be great for the future.